tools/hangar/frontend/bindings/github.com/fleetdm/fleet/tools/hangar/services/settingsservice.ts

```
@@ -28,10 +41,19 @@ export function DetectFleetConfig(repo: string): $CancellablePromise<string> {
-        return $$createType0($result);
+        return $$createType1($result);
     });
 }
 
+/**
+ * LanIP is the host's current LAN IPv4, for the surfaces that show an address
+ * another device has to dial (SCEP enrollment URLs, the python file server).
+ * Empty means "unknown" — see netinfo.LanIP.
+ */
+export function LanIP(): $CancellablePromise<string> {
+    return $Call.ByID(421620667);
```

ESLint

[error] Delete `··`

(prettier/prettier)